Boa.Constrictor.Playwright/CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+This file documents all notable changes to the Boa.Constrictor.Xunit project and its unit tests for each NuGet package release.
+
+Its format is based on Keep a Changelog, and this project adheres to Semantic Versioning.
+
+## [Unreleased]
```

Comment:
Make this version `4.0.0` with today's date. GitHub Actions should automatically publish this package once this PR is merged.